src/classes/VirtualTryOnFrames.ts

```diff
@@ -10,11 +10,6 @@ import { getVTOFrames } from './Frames';
 export const virtualTryOnFrames = async ({ sku }: VirtualTryOnFramesProps): Promise<any> => {
     try {
         const userProfile = await Auth.getUserProfile();
-        const userVTOFrames = await getVTOFrames({ sku: sku }) as UserVTOFrames;
```

Comment:
If the user doesn't have an avatar yet, lets not bother with attempting to retrieve frames.

Reply:
This is already handled, if the user don't have a avatar yet it will render a modal which will tell the use to get the app and make a avatar. So, it can't come here if it doesn't have a avatar

Reply:
If a user of the sdk uses this function only, it wouldn't be handled. Thats why this change is necessary.